src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1147,6 +1205,17 @@ private AuthenticationResult acquireTokenAfterValidation(CallbackHandler callbac
             AuthenticationResult result = null;
             request.setVersion(getVersionName());
             request.setBrokerAccountName(request.getLoginHint());
+            
+            //check if App gives permission to AccountManager
+            try
```

Comment:
Well, I know why it got hit in my project. I comment off the verifyManifestPermissions in canSwitchToBroker() BrokerProxy.java.